Examples/Framework/include/ActsExamples/EventData/SpacePointContainer.hpp

```diff
@@ -0,0 +1,118 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/EventData/Holders.hpp"
+
+#include "ActsExamples/EventData/SimSpacePoint.hpp"
+
+// memory backend is a vector of pointer of SimSpacePoint
+
+namespace ActsExamples {
+  
+  template<typename collection_t>
+  class SpacePointContainer {
```

Comment:
Technicality, if it is meant to be CRTP pattern don't you need inheritance from `public Acts:SpacePointContainer<ActsExamples::SpacePointContainer>` ?

In any case, I am not sure why this class is template at all? Naively I would think this would be a concrete implementation class.